parsers/nmparser/parse_lst_file.go

```diff
@@ -374,7 +387,7 @@ func ParseLstEstimationFile(lines []string) ModelOutput {
 		case strings.Contains(line, "0MINIMIZATION SUCCESSFUL"):
 			runHeuristics.MinimizationSuccessful = HeuristicTrue
 		case strings.Contains(line, "GRADIENT:"):
```

Comment:
can you please add a piece of doc as to why this is needed. I believe this is because it looks like for larger number of params it can run onto a new line and thus you need to chase that down, but these "deviations" from the simple matching should get noted

Reply:
Initially, this is the style I used (on purpose) for string matching, thinking I did not want to "fail to match" because of whitespace or other irrelevant characters, especially WRT to different control streams and different versions of NONMEM.  I've now seen that the lst text is pretty consistent, so an "equal-to" test would probably be fine.